VDUBStep/Classes/VDUBStore.h

```diff
@@ -13,9 +13,8 @@
 + (VDUBStore *)sharedInstance;
 
 + (NSManagedObjectContext *)mainContext;
++ (NSManagedObjectContext *)backgroundContext;
 
 - (void)setupCoreDataStackWithURL:(NSURL *)storeURL;
 
-- (NSManagedObjectContext *)mainContext;
```

Comment:
Also I may be a little slow to answer from now on as we are on the ship, though we do have unlimited internet so that helps!  

—> Kendall

> On Aug 23, 2015, at 9:54 PM, Kendall Gelner <Kendall.Gelner@KiGiSoftware.com> wrote:
> 
> Lately I’ve been preferring class-level stuff also when an instance doesn’t matter, not sure why but I say go for it!  We are usually tuned in the same way somehow.  The rest of the world will follow eventually.
> 
> —> Kendall
> 
>> On Aug 21, 2015, at 4:26 PM, Bruce Geerdes <notifications@github.com <mailto:notifications@github.com>> wrote:
>> 
>> In VDUBStep/Classes/VDUBStore.h <https://github.com/bgeerdes/vdubstep/pull/2#discussion_r37684123>:
>> 
>> >  
>> >  - (void)setupCoreDataStackWithURL:(NSURL *)storeURL;
>> >  
>> > -- (NSManagedObjectContext *)mainContext;
>> Wondering if I should go ahead and make setupCoreDataStack class-level and remove sharedInstance from the public interface.
>> 
>> —
>> Reply to this email directly or view it on GitHub <https://github.com/bgeerdes/vdubstep/pull/2/files#r37684123>.
>> 
>